smallrye-reactive-messaging-provider/src/test/java/io/smallrye/reactive/messaging/inject/EmitterInjectionTest.java

```diff
@@ -137,21 +167,24 @@ public void testWithTwoEmittersConnectedToOneProcessor() {
         }
 
         public void run() {
-            emitter.send("a").send("b").send("c").complete();
+            emitter.send("a");
+            emitter.send("b");
+            emitter.send("c");
+            emitter.complete();
         }
 
         @Incoming("foo")
-        public void consume(String s) {
+        public void consume(final String s) {
             list.add(s);
         }
     }
 
     @ApplicationScoped
     public static class MyBeanEmittingPayloadsUsingStream {
         @Inject
-        @Stream("foo")
+        @Channel("foo")
```

Comment:
This test is about testing the legacy Stream annotation, do don't update to @Channel.

Reply:
done